module/VuFind/src/VuFind/AjaxHandler/GetRequestGroupPickupLocations.php

```diff
@@ -53,6 +53,7 @@ public function handleRequest(Params $params)
     {
         $this->disableSessionWrites();  // avoid session write timing bug
         $id = $params->fromQuery('id');
+        $itemId = $params->fromQuery('itemId');
```

Comment:
I don't see any corresponding code to pass this new value to the handler. Is it already getting passed but not being processed, or is something missing? As with the cache trait improvements, please let me know if you think this is something that should be merged independently ahead of the driver itself.